### ISSUE TITLE
Consider adding a hash history mode

### ISSUE THREAD

**Issue by charlieduong94** [charlieduong94/marko-path-router#3]
This might be useful for anyone that wants to serve a single page app via a CDN of some sort, but does not want to set up a proxy that would serve the app on all routes that do not point to static assets.

**Comment by cking** [comment]
i am currently in need to hopefully use something based on hash history. it is a client side only application (only images, css and js is served from a webserver). right now im running a SPA proxy in front of it, just to delegate the routes. it would be nice to not have to use one. :)

**Comment by charlieduong94** [comment]
Hello @cking, I'll try to get hash history implemented this weekend.

**Comment by cking** [comment]
That would be highly appreciated

**Comment by charlieduong94** [comment]
@cking Hash history mode has landed in `0.6.0`. I made a branch of the demo to display it's usage [here](https://github.com/charlieduong94/marko-path-router-demo/tree/hash-history). I tried to make the transition from standard browser history to hash history pretty minimal. I recommend checking out that branch's [diff](https://github.com/charlieduong94/marko-path-router-demo/commit/1a934a3d8bffddfc00e322e2512cb66c92a4b42d) to see what changes needed to be made. Let me know if encounter any issues.
